src/main/java/org/vandeseer/easytable/TableDrawer.java

```diff
@@ -123,9 +123,8 @@ protected void drawWithFunction(Point2D.Float startingPoint, TableDrawerFunction
 
     protected void drawBackgroundColorAndCellContent(Point2D.Float start, AbstractCell cell) throws IOException {
 
-        final float rowHeight = cell.getRow().getHeight();
-        final float height = cell.getHeight() > rowHeight ? cell.getHeight() : rowHeight;
-        final float y = cell.getHeight() > rowHeight ? start.y + rowHeight - cell.getHeight() : start.y;
+        final float height = cell.getRow().getHeight();
+        final float y = start.y;
```

Comment:
This particular part was not correct: It breaks row spanning. :wink: 

I changed it back in `develop` now. Unfortunately one doesn't realize immediately by only running the unit tests, only by comparing the output PDFs visually ...